src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
+
+.form-big {
+  position: relative;
+  margin: 20px 8px;
+}
+
+.form-small {
+  position: relative;
+  margin: 0 8px 8px;
+}
+
+.searchBar-big {
+  box-sizing: border-box;
+  font-family: Avenir, serif;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.icon_big {
+  position: absolute;
+  left: 26px;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.searchBar-small {
+  box-sizing: border-box;
+  font-family: Avenir, serif;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.icon_small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.searchBar-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
```

Comment:
you can make a class for both search bar and use it, where the searchbars have the same values